parsers/parsers.go

```diff
@@ -36,6 +45,35 @@ func (e *Empty) DoFromHuman(string) string {
 	return "Not Yet Implemented"
 }
 
-var ErrTooLarge error = errors.New("Too Beaucoup")
-var ErrTooSmall error = errors.New("Number too small")
-var ErrNotANumber error = errors.New("Not a Number")
+// IsMachineNumber validates that a number:
+// Contains only digits
+// The first digit must be 1-9 when >= 10
+func isMachineNumber(s string) bool {
+	match, _ := regexp.MatchString(`^[0-9]$|^[1-9][0-9]+$`, s)
+	if match {
+		return true
+	}
+	return false
```

Comment:
since `match` is a bool,  we can just return that
```suggestion
	match, _ := regexp.MatchString(`^[0-9]$|^[1-9][0-9]+$`, s)
	return match
```